pkgs/applications/audio/openutau/deps.json

```diff
@@ -0,0 +1,1207 @@
+[
+  {
+    "pname": "AsyncIO",
+    "version": "0.1.69",
+    "hash": "sha256-JQKq/U71NQTfPuUqj7z5bALe+d7G1o3GcI8kvVDxy6o="
+  },
```

Comment:
I haven't measured how this impacts filesize or git object size.

Just looking at line count though, the lockfiles are 5x longer. So a lockfile that had ~200 lines is now ~1000 lines.

Personally, I don't see this as an issue. Especially if actual filesize is relatively unaffected. But it should probably be discussed at least.

Do we want "normal" pretty formatting or do we want one line per package?

Reply:
I don't see why we wouldn't want it formatted. All other JSON lockfiles in nixpkgs are formatted, so if someone wants this unformatted we wouldn't be the only place to change.

Reply:
This has no impact on entropy which is the only thing we care about w.r.t. git.